.github/workflows/publish.yaml

```diff
@@ -130,3 +130,23 @@ jobs:
 
     - name: Pact record-release
       run: make record-release
+
+    - name: get previous release tag
+      id: previous-release
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const {
+            data: [latest, previous],
+          } = await github.rest.repos.listReleases({
+            ...context.repo,
+            per_page: 2,
+            page: 1,
+          });
+          if (!previous) return;
+          core.setOutput("sha", previous.tag_name);
+
+    - if: steps.previous-release.outputs.sha
+      env:
+        PACT_VERSION: ${{ steps.previous-release.outputs.sha }}
+      run: make record-support-ended
```

Comment:
can we strategize on this bit? i'm good with the other changes, but many production customers will not be running the latest release, so we will want to maintain support for at least a representative sample of versions